x-pack/plugins/security_solution/common/endpoint/data_generators/endpoint_action_generator.ts

```diff
@@ -45,6 +45,7 @@ export class EndpointActionGenerator extends BaseDataGenerator {
         agent: {
           id: [this.seededUUIDv4()],
         },
+        hosts: undefined,
```

Comment:
Afaik this generator is not used in automated response actions. And only these actions contain hosts, otherwise it's calculated after action and response are fetched.